tests/cli/dorq/test_repos.py

```diff
@@ -110,14 +111,10 @@ def test_passing_errors(monkeypatch, exc):
                 # Given
                 config = "<config sentinel>"
 
-                # Prevent FS access
-                monkeypatch.setattr(_config, "read_config", Mock())
+                def raise_exc(*args, **kwargs):
+                    raise exc
 
-                runtime = Mock()
-                runtime.get_all_workflow_runs_status.side_effect = exc
-                monkeypatch.setattr(
-                    _factory, "build_runtime_from_config", Mock(return_value=runtime)
-                )
+                monkeypatch.setattr(sdk, "list_workflow_runs", raise_exc)
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches and the blocking or non-blocking decorations to further communicate your intent.  -->
**suggestion (non-blocking):** I dont have any strong feelings, but I think this could be:
 ‏
```suggestion
                monkeypatch.setattr(sdk, "list_workflow_runs", Mock(side_effect=exc))
```